app/services/work_ocr_creator_remover.rb

```diff
@@ -34,6 +34,6 @@ def image_assets
       members.
       where(type: 'Asset').
       order(:position).
-      select { |m| m.content_type.start_with?("image/") }
+      select { |m| m&.content_type&.start_with?("image/") }
```

Comment:
Adding this in here as well - just want it to fail gracefully if for whatever reason an asset doesn't have a content_type defined on it.